### PR TITLE
edit and delete comments for user only

### PR DESCRIPTION
**Pull request by therese28**
Pull request for user only to edit and delete comments (currently no difference for admins and other users)

### CODE REVIEW

**Comment by therese28** [comment]
https://www.loom.com/share/596e9300f26f4d9f88c4dc7d2a532281

**Comment by RipNasty** [comment]
Worked through this with Therese.  Incorporated changes based on Nick's help, looked through and checked for indentation errors.

**Comment by Chepenik** [comment]
Caught a console.log that needs to be removed but besides that looks great

**Comment by AngeC22** [comment]
looks good

File: client/src/components/CommentTile.js [hunk 1]
```diff
@@ -12,11 +12,15 @@ const CommentTile = ({ comment, handleCommentDelete}) => {
...
       <p>Rating: {rating}</p>
       <p>{text}</p>
+      {currentUser && currentUser.id === comment.userId && (
+      <> 
       <button type="delete" onClick={handleClickDelete}>Delete</button>
       <br></br>
       <Link to={`/comments/${id}/edit`} className="social-media-item">
         <p>Edit Your Comment</p>
       </Link>
+      </>
```
Comment (nwalberts) [inline client/src/components/CommentTile.js:22]: indent in lines 16 to 22

File: client/src/components/EditForm.js [hunk 2]
```diff
@@ -39,17 +39,16 @@ const EditForm = (props) => {
...
-            body: JSON.stringify({ comment: updatedCommentData }),
-            }
+            const response = await fetch(
+                `/api/v1/comments/${commentId}`,
+                {
+                method: "PATCH",
+                headers: {
+                    "Content-Type": "application/json",
+                },
+                body: JSON.stringify({ comment: updatedCommentData }),
```
Comment (nwalberts) [inline client/src/components/EditForm.js:50]: fix indentation. 

- the url should be on line 43 with fetch
- line 45 should also be at the end of line 43 (see previous lessons)

**Pull request merged**